x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx

```diff
@@ -94,6 +94,7 @@ export interface TGridStandaloneProps {
   filters: Filter[];
   footerText: React.ReactNode;
   filterStatus: AlertStatus;
+  hasAlertsPermissions: (featureId: string) => boolean;
```

Comment:
This is optional in t_grid/body but not here, just making sure that's intentional?

Reply:
yes, it is because the standalone need to have it but the integrated does not need it that's why it is optional